xendit_pos/models/xendit_client.py

```diff
@@ -71,6 +72,8 @@ def create_invoice(self, payment_method, data):
                 res.status_code
             )
 
+        # generate qrcode
+        response['qrcode_image'] = self.qrCode.renderQrcode(response['invoice_url'])
```

Comment:
interesting, what is the difference between the old version from this one? @andykim

Reply:
@yanoandri In the current version we generate the QRcode from a third-party service https://github.com/xendit/odoo-pos-module/blob/d8b5a7aa8bbc9bccd85290b2542d3c134c6ae66d/xendit_pos/static/src/js/payment_xendit_pos.js#L199